src/main/java/Task.java

```diff
@@ -0,0 +1,23 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatus() {
+        String icon = isDone ? "X" : " ";
+        String status= "[" + icon + "] "+ description;
```

Comment:
Any reason why you are returning the entire print line? I'm asking because `getStatus()` should only return the value of `status`. I recommend extracting it out and override the toString() function as part of your print statement instead.